misk/web/tabs/loader/src/containers/LoaderContainer.tsx

```diff
@@ -0,0 +1,79 @@
+import { IconName, IconNames } from "@blueprintjs/icons"
+import * as React from "react"
+import { connect } from "react-redux"
+import { Route, Switch } from "react-router"
+import styled from "styled-components"
+import { IAppState } from ".."
+import { dispatchAdminTabs } from "../actions"
+import { NoMatchComponent, ScriptComponent } from "../components"
+
+interface ITabProps {
+  adminTabs: IAdminTabs
+  loading: boolean
+  error: any
+  getTabs: any
+}
+
+export interface ILoaderState {
+  adminTabs: IAdminTabs
+}
+
+export interface IAdminTabs {
+  [key:string]: IAdminTab
+  toJS: any
+}
+
+export interface IAdminTab {
+  icon: IconName
+  name: string
+  slug: string
+  url_path_prefix: string
```

Comment:
Is there a good reason to use an underscored name instead of camel case?

Reply:
This was a decision made to match variables against the request code in the Misk Kotlin server so that the variables stand out as ones that have come from a request.